.env.example

```
@@ -1,47 +1,46 @@
 # 🌐 Frontend
+FRONTEND_PORT=8015 # Don't forget to update CSRF_TRUSTED_ORIGINS in backend and FRONTEND_URL in backend and ORIGIN in frontend if you make changes to this.
 PUBLIC_SERVER_URL=http://server:8000 # PLEASE DON'T CHANGE :) - Should be the service name of the backend with port 8000, even if you change the port in the backend service. Only change if you are using a custom more complex setup.
 ORIGIN=http://localhost:8015
 BODY_SIZE_LIMIT=Infinity
-FRONTEND_PORT=8015
 
 # 🐘 PostgreSQL Database
-PGHOST=db
-POSTGRES_DB=database
-POSTGRES_USER=adventure
+PGHOST=db # Supposed to match the host of the postgis database
+POSTGRES_DB=adventureloga
```

I think there is a typo here.
```suggestion
POSTGRES_DB=adventurelog
```